packages/ai/src/ai/account/file_store.py

```
@@ -350,6 +350,72 @@ async def mkdir(self, path: str) -> None:
+
+        # Ignore the dirmarker sentinel when checking emptiness
+        non_marker = [f for f in all_files if not f.endswith('/' + DIR_MARKER)]
+
+        if not recursive and non_marker:
+            raise StorageError(f'Directory not empty: {path}')
+
+        for f in all_files:
+            try:
+                await self._store.delete_file(f)
+            except StorageError:
+                pass
```

Ruff (0.15.10)

[warning] Use `contextlib.suppress(StorageError)` instead of `try`-`except`-`pass`

Replace `try`-`except`-`pass` with `with contextlib.suppress(StorageError): ...`

(SIM105)